lib/utils/socketChainHelper.js

```diff
@@ -53,6 +53,15 @@ const processServerResponse = toString => function processServerResponseHandler(
 		);
 	}
 
+	if (res.errorType === 'appRunning') {
+		let msg = 'App is still running';
+
+		if (data.type === 'press') {
+			msg += `\nMaximum amount of key presses ${data.repeat} reached. Condition was not satisfied`;
```

Comment:
OK, from the code I assume the only possible case when `appRunning` error can occur is for line "Press [] until application exits"... Is that so?

Also, all texts must be moved to `texts.js` file in `lib` folder.

Reply:
When `suitest.application().hasExited()` we have just `appRunning` error but in case of press e.g. 
``` suitest.assert.press(suitest.VRC.UP).until(
			suitest.application().hasExited()
		).repeat(5).interval(2000);
 ```
we also add `conditionNotSatisfied` error

Reply:
OK, in that case we should take into account that "application has exited" is not the only chain that can be passed to "until". And "until" can be chained to other subjects then "press".

Reply:
I think this `conditionNotSatisfied` errorType is not very accurate. It was added on FE for press line only, but was not updated when we added browser lines (click, send text etc) (possible FE bug). If we really want this message `Maximum amount of key presses ${data.repeat} reached. Condition was not satisfied`, I suggest we simply add it to all failed testLine chains with `until` modifier and rephrase it to something like `Maximum amount of iterations was made. Condition was not satisfied`?